BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Internal/Sampler.cs

```diff
@@ -45,6 +45,7 @@ public void Start()
         public bool Sampled(Span span)
         {
             var p = Probability;
+            span.SetAttribute("bugsnag.sampling.p", p);
```

Comment:
I think technically this step would only be needed if the span is sampled? Otherwise the span is just going to be thrown away.

Reply:
yes right 👍